pkg/controller/syncsetinstance/syncsetinstance_controller.go

```diff
@@ -71,8 +73,29 @@ const (
 
 var (
 	applyTempFileMatcher = regexp.MustCompile(`/apply-\S*(\s|$)`)
+
+	metricTimeToApplySyncset = prometheus.NewHistogram(
+		prometheus.HistogramOpts{
+			Name:    "hive_syncset_apply_duration_seconds",
+			Help:    "Time to apply syncset",
+			Buckets: []float64{5, 10, 30, 60, 300, 600, 1800, 3600},
+		},
+	)
+	metricTimeToApplySelectorSyncset = prometheus.NewHistogramVec(
+		prometheus.HistogramOpts{
+			Name:    "hive_selector_syncset_apply_duration_seconds",
```

Comment:
Suggest hive_selectorsyncset_apply_duration_seconds to better match the name of the resource.